### ISSUE TITLE
compatibility with ansible-lint 4

### ISSUE THREAD

**Issue by themr0c** [geerlingguy/ansible-role-docker#132]
Using ansible-lint 4.1.0, molecule test will fail on lint:

````
    [306] Shells that use pipes should set the pipefail option
    [...]/geerlingguy.docker/tasks/setup-Debian.yml:28
````

**Comment by maciej-wawrzynczuk** [comment]
It looks like fixed with 5afc0f8.

**Comment by geerlingguy** [comment]
Yes, this is fixed. Thanks!
